Tests/MeiliSearchIntegrationTests/KeysTests.swift

```diff
@@ -5,69 +5,93 @@ import Foundation
 // swiftlint:disable force_try
 class KeysTests: XCTestCase {
   private var client: MeiliSearch!
-  private var key: String = ""
   private var session: URLSessionProtocol!
 
   // MARK: Setup
 
   override func setUp() {
     super.setUp()
+
     session = URLSession(configuration: .ephemeral)
     client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey", session: session)
-    let keyExpectation = XCTestExpectation(description: "Get all keys")
 
-    self.client.getKeys { result in
+    // remove all keys to keep a clean state
+    self.client.getKeys(params: KeysQuery(limit: 100, offset: 0)) { result in
       switch result {
       case .success(let keys):
-        if keys.results.count > 0 {
-          let key = keys.results.first
-          if let firstKey: Key = key {
-            self.key = firstKey.key
+        keys.results.forEach {
+          self.client.deleteKey(key: $0.uid) { result in
+            switch result {
+            case .success:
+              ()
```

Comment:
is this the same as `keyExpectation.fulfill()` ?

Reply:
When we define a `switch`, we should provide an exhaustive list of possibilities, such as success and failure. The problem is that Swift requires me to provide at least a statement when defining a `case.` Since this method is meant to be a setup and I'm not interested in the success, I could provide the `()` empty function, but I would like to know if there is another way to handle that.

Reply:
Even if you are not interested in the success is `.fulfill()` not used to stop the wait of the expectation?

Reply:
yeah, but that's the point, this code is written inside of the setup(), so it's not meant to be a "test" so in other words, no expectation should be needed 😬